FreeRTOS/FreeRTOS
# Remove dependency of CBMC on Patches

Description
-----------
This PR removes dependency on patches for CBMC checks.
It also adds a macro '_static' to the source which can be defined away to nothing by the CBMC checks. In `FreeRTOSIPConfigDefaults.h` this macro is defined to the standard `C` `static`.

For easy review, below are the commits bunched together in groups:
All changes to FreeRTOS+TCP [here](https://github.com/FreeRTOS/FreeRTOS/pull/181/files/0caa0f0f8dd260d8f6de1b55e71cef9e9cab7d1a..baa8e427e2ae947bfa0295ecd0a7b492c2415eed).
All CBMC changes [here](https://github.com/FreeRTOS/FreeRTOS/pull/181/files/baa8e427e2ae947bfa0295ecd0a7b492c2415eed..d2164820a580fd680b74e2c239b04bf2b6550e36).

Test Steps
-----------
Not Applicable

Related Issue
-----------
Not Applicable


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
